src/edfi-performance-test/edfi_performance_test/api/client/course_offering.py

```diff
@@ -15,8 +15,9 @@ class CourseOfferingClient(EdFiAPIClient):
 
     def create_with_dependencies(self, **kwargs):
         school_id = kwargs.pop("schoolId", SchoolClient.shared_elementary_school_id())
-        school_year = kwargs.get("schoolYear", 2014)
-        session_reference = self.session_client.create_with_dependencies(
+        school_year = kwargs.pop("schoolYear", 2014)
```

Comment:
`pop` instead of `get`: without the pop, the `schoolYear` ends up directly in the "root" of the course_offering, which is not correct. ODS/API ignores that overposting, but Meadowlark does not.